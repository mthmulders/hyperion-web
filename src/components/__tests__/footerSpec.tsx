import * as React from 'react';

import { shallow } from 'enzyme';

import Footer from '../footer';

describe('<Footer />', () => {
    it('should render', () => {
        // Act
        const wrapper = shallow(<Footer />);

        // Assert
        expect(wrapper).toMatchSnapshot();
    });
});
