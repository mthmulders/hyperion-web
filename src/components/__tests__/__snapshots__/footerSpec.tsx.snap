// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span
      className="text-muted"
    >
      Â© 2016 - 2019 Maarten Mulders. Source code is available at 
      <a
        href="https://github.com/mthmulders/hyperion-web"
        target="_blank"
      >
        GitHub
      </a>
      .
    </span>
  </div>
</footer>
`;
